### ISSUE TITLE
automate/add to process bumping metallb in minikbue ?

### ISSUE THREAD

**Issue by medyagh** [metallb/metallb#1862]
hello metallb 

I am minikube maintainer and I do love metallb ! and our users love it too, however recently we noticed the addon was broken in minikube https://github.com/kubernetes/minikube/issues/16053  because of moving to quay.io https://github.com/metallb/metallb/pull/784

in minikube github action we have about 10 auto-updater github workflows that updates different things automitcally, I would like to know get your inisght how we can make this happen, so our users use the lates and greatest metallb on each minikube release.

I personally tired to just bump the version in this file (https://github.com/medyagh/minikube/blob/397eeac5bf00305495b3f3a8df711caf2f075a09/pkg/minikube/assets/addons.go#L545-L548)
```
}, false, "metallb", "3rd party (MetalLB)", "", "", map[string]string{
		"Speaker":    "metallb/speaker:v0.9.6@sha256:d34f613a88bdeba10f96a50554fb6a4a659cbc99270b1adc93e3e21fa7f6c1ad",
		"Controller": "metallb/controller:v0.9.9@sha256:cb34fded834feea570ec26159d2030e364ec98fc58f75f555a7acf1822791604",
	
```
to "v0.13.9" but that didnt work it seems like we also have to update the yaml files too ?

[deploy/addons/ambassador/ambassador-operator-crds.yaml.tmpl](https://github.com/kubernetes/minikube/tree/master/deploy/addons/ambassador)  ? 

I like to pick your brains what is the smoothest way to update ?

**Comment by fedepaol** [comment]
hi! Thanks for reaching out.
Bumping the image to latest won't work for a couple of reasons. One is, parameters changed and you must have the manifests aligned, the other is, we pivoted our configuration from configmap to crds.

I see you solved the contingent problem with https://github.com/kubernetes/minikube/pull/16056 by sticking to the same version.

Then, for the "keeping minikube aligned to metallb" part, all you need for the deployment is https://github.com/metallb/metallb/blob/main/config/manifests/metallb-native.yaml

I don't suggest templating and changing only the image because as I said, there might be dependencies such as parameters, volumes and so on that change across releases. 
When bumping to one of the new releases, https://github.com/kubernetes/minikube/blob/master/deploy/addons/metallb/metallb-config.yaml.tmpl will need to be changed too in order to reflect the new configuration methods.

So, TL;DR: I'd look for a new tag and use that all in one manifests to keep in line. I can try to push a pr but not sure when (I guess it's not critical today). 
If you have any other questions, feel free to reach us on #metallb-dev on the k8s slack

**Comment by medyagh** [comment]
sounds good thanks for clarifying it, does metallb has a release process to follow step by step ? maybe one post-release step would be "bump metallb in other projects, minikube, etc..."

**Comment by fedepaol** [comment]
> sounds good thanks for clarifying it, does metallb has a release process to follow step by step ? maybe one post-release step would be "bump metallb in other projects, minikube, etc..."

yep, it's documented here https://metallb.universe.tf/community/release-process/ but I'd rather help to automate it instead of doing it manually.

**Comment by fedepaol** [comment]
not stale
